packages/compass/webpack.config.js

```diff
@@ -98,16 +97,41 @@ module.exports = (_env, args) => {
     HADRON_METRICS_SEGMENT_HOST: null,
   };
 
+  const resolve = {
+    alias: {
+      /** @type {false} */
+      'ampersand-sync': false,
```

Comment:
Maybe it would make sense to do it for the shared configuration? And additionally maybe worth adding a note why we are doing this?

Reply:
> shared configuration

is it visible enough there? what do we need to bundle in addition to compass with the shared config?

Reply:
I think it will be visible enough in both places. Plugins are bundled with shared config, but thinking about it, there are probably no plugins used by DE that will have ampersand-sync (or any models in general) in them, so probably an overkill to move it 👍

Reply:
Moved :)